shell/utils/create-yaml.js

```diff
@@ -87,7 +87,8 @@ export function createYaml(
   depth = 0,
   path = '',
   rootType = null,
-  dataOptions = {}
+  dataOptions = {},
+  commentFieldsOptions = null
```

Comment:
Is it worth to add `commentFieldsOptions` arg to `createYamlWithOptions` function as well?